pydpu/proto/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Add proto dir to PYTHONPATH since protoc does not create relative imports
+# See this for more info https://github.com/protocolbuffers/protobuf/issues/1491
```

Comment:
this is cool find!! thanks!! I was struggling why my imports not working

Reply:
kinda sad to see that they're not planning to add support for relative imports. I slightly prefer this workaround to the one where we search-replace all generated files to make them relative, since this way we are not touching generated files.